pkg/oceanbase-sdk/model/server.go

```diff
@@ -31,3 +31,24 @@ type OBServer struct {
 	StartServiceTime int64  `json:"start_service_time" db:"start_service_time"`
 	BuildVersion     string `json:"build_version" db:"build_version"`
 }
+
+// GVOBServer shows the usage info of the server
+type GVOBServer struct {
```

Comment:
Maybe this struct can be changed to a meaningful name like OBServerResource

Reply:
OK